XcodeServerSDK/Utils/HTTPUtils.swift

```diff
@@ -53,13 +51,13 @@ public class HTTP {
                         
                     case let s where s.rangeOfString("application/json") != nil:
                         
-                        let (json: AnyObject!, error) = JSON.parse(data)
-                        let headers = httpResponse.allHeaderFields
+                        let (json, error) = JSON.parse(data!)
```

Comment:
That what I thought but...
![zrzut ekranu 2015-06-16 o 21 51 11](https://cloud.githubusercontent.com/assets/4522407/8192965/eaab291a-1471-11e5-9351-ec9684077b1c.png)